kiwix.yml

```diff
@@ -510,9 +510,9 @@ all:
     version: "2015-03-16"
     size: "2949838231"
     url: "http://download.kiwix.org/portable/other/kiwix-0.9+bil-tunisia_ar_all_2015-03.zip"
-    name: "Bil Tunisia -- Bil Tunisia videos"
+    name: "فيديو-بالتونسي"
     language: "ara"
-    description: "Bil Tunisia videos"
+    description: "مقاطع فيديو باللغه التونسيه"
```

Comment:
True. It's half/half.
Like the same person doing half the phrase in French and half in Arabic…
But let's say this package is mainly targeted to people speaking both languages, so we can either keep the metadata in French (but then make it better) or in Arabic.